src/olympia/devhub/templates/devhub/addons/onboarding_subscription.html

```diff
@@ -18,20 +18,31 @@
     <h2>{{ _('Set Up Payment') }}</h2>
 
     {% if stripe_checkout_completed %}
-    <p>
-      {% trans %}
-      You're done!
-      {% endtrans %}
-    </p>
```

Comment:
Why not keeping this sentence like this? It's displayed in both cases below.

Reply:
it added some non-trivial logic for a few lines of text - it's displayed in cases when `already_promoted` or `new_version_number` (now I write this... it's not complicated though) but not for new promotions where we don't have the version number (i.e. it failed).  The PRD comments from Jorge seemed like he wanted it as a single paragraph anyway so I simplified it.  Can keep it separate if you think that's best?

Reply:
not a big deal